### PR TITLE
clean up of RawText to text.components (#8)

### LINKED ISSUE #8: Add Translations
We are using RawText where we should be using Text components and there are a lot of other places we are just passing untranslated text into buttons and more

Here are a few places that need updating from the wallet merge
- PinModal
- NativeImport
- NativePassword
- NativePasswordRequired
- NativeSeed
- NativeStart
- NativeSuccess
- NativeTestPhrase
- SelectModal

### PR DESCRIPTION
**Pull request by 420coupe**
## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [x] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)
Closes #8 

## Description

Updated RawText to Text variables stored in messages-en.json. Ideally this will be the one file sent out for translations. 

## Testing

1. Pull branch locally and run `yarn` to install new deps
2. Run `yarn dev` this will launch a local copy on http://localhost:3000/
3. Verify all changes worked and nothing breaks.

### CODE REVIEW

File: src/assets/translations/messages-en.json [hunk 1]
```diff
@@ -61,6 +61,57 @@
...
+      },
+      "nTest": {
+        "header": "Let's practice your secret phrase",
+        "body": "Which of these is the",
+        "body2": " word",
+        "body3": "in your secret phrase.",
+        "button": "Next"
+      }
+    },
+    "sModal": {
```
Comment (0xdef1cafe) [inline src/assets/translations/messages-en.json:110]: what does `sModal` denote? would this be better named `sendModal`?

File: src/assets/translations/messages-en.json [hunk 2]
```diff
@@ -61,6 +61,57 @@
...
     }
   },
+  "wProvider": {
+    "keepKey": {
+      "header": "Enter Your Pin",
+      "body": "Use PIN layout shown on your device to find the location to press on the pin pad.",
+      "button": "Unlock"
+    },
+    "shapeShift": {
+      "nImport": {
```
Comment (0xdef1cafe) [inline src/assets/translations/messages-en.json:71]: what does the `n` prefix of these keys denote? don't be afraid to be explicit - i'm not sure what this means.

File: src/assets/translations/messages-en.json [hunk 3]
```diff
@@ -61,6 +61,57 @@
...
+        "button": "Next"
+      },
+      "nPasswd": {
+        "header": "Enter your password",
+        "body": "Enter a password to encrypt your wallet. In order to securely store your keys we will encrypt them, choose a string password you can remember.",
+        "button": "Next"
+      },
+      "nPassReq": {
+        "header": "Enter your password",
+        "body": "Enter a password to encrypt your wallet. In order to securely store your keys we will encrypt them, choose a string password you can remember.",
```
Comment (0xdef1cafe) [inline src/assets/translations/messages-en.json:83]:
```suggestion
        "body": "Enter your native wallet password to decrypt the wallet stored in your browser.",
```

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

File: src/assets/translations/messages-en.json (hunk 3 above)
Comment (0xdef1cafe) [reply src/assets/translations/messages-en.json:83]: this is actually a bug, if we change it here we can squash it now :)

**Comment by 0xdef1cafe** [comment]
ty for your PR - just a few requested changes, otherwise looks great.

File: src/assets/translations/messages-en.json (hunk 1 above)
Comment (420coupe) [reply src/assets/translations/messages-en.json:110]: sModal is an abbreviation for SelectModal.tsx

File: src/assets/translations/messages-en.json (hunk 2 above)
Comment (420coupe) [reply src/assets/translations/messages-en.json:71]: 'Native' - just abbreviating for sake of typing out the variables

**Review by bootsboogie — APPROVED** [review]
Looks good to me. Thanks for contributing! And updating those chakra Texts even though you did not add them.

**Comment by jjfarrow** [comment]
Thanks for contributing!

About the abbreviations - I notice that other places in the code we are using "SelectModal", not "sModal", and same with the other abbreviations brought in here. My personal preference is full names, as I think it makes the code more readable to new contributors, and we want to keep this friendly for new folks.

I think this is what @0xdef1cafe was saying with his comments. @bootsboogie what are your thoughts about that? I see you approved the PR, so maybe not such a thing for you? However we decide, I think it makes sense to keep it consistent across the repo.

**Comment by notatooyou** [comment]
Thanks for contributing!

About the abbreviations - I notice that other places in the code we are using "SelectModal", not "sModal", and same with the other abbreviations brought in here. My personal preference is full names, as I think it makes the code more readable to new contributors, and we want to keep this friendly for new folks.

I think this is what @0xdef1cafe was saying with his comments. @bootsboogie what are your thoughts about that? I see you approved the PR, so maybe not such a thing for you? However we decide, I think it makes sense to keep it consistent across the repo.

**Review by 0xdef1cafe — APPROVED** [review]

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**